src/components/ui/ListSection/ListSection.styled.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+const ListSection = styled.section`
```

Comment:
Something I've noticed about this particular element is that you're positioning the svg with absolute/relative, when you could probably do so with `display: flex;` on the <li> containing both the <a> and <svg> tags.

Sidenote: you're not actually using absolute/relative. The `position: absolute;` on your <svg> tag only seems to do anything because it references the next possible parent element within its stacking context. The `position="relative"` attribute on your <a> tag doesn't work, because it doesn't exist as an attribute in HTML I'm pretty sure. The `margin` values in your <svg> element seem to do the most heavy-lifting in regards to the positioning of the arrow, but there's a whole lot going on currently that's ultimately not necessary.